en/integrations/plugin-packs/procedures/applications-databases-elasticsearch.md

```diff
@@ -1,38 +1,141 @@
+| :---------------- | :---------------------------------------------------- | :-----  |
+| display           | Name of the Elasticsearch node                        |         |
+| indices_count     | Number of indices in the node                         |         |
+| heap_used_percent | Percentage of memory currently in use by the heap     |    %    |
+| heap_used_in_bytes| Amount of memory available for use by the heap        |  bytes  |
+| heap_max_in_bytes | Maximum amount of memory available for use by the heap|  bytes  |
+| available_in_bytes| Total number of bytes available                       |  bytes  |
+| total_in_bytes    | Total size of all file stores                         |  bytes  |
+| docs_count        | Number of documents inside the indice                 |         |
+| size_in_bytes     | Total size of all shards assigned to the node         |  bytes  |
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prerequisites 
+
+In order to monitor an Elasticsearch cluster, it must be prepared acccording to Elasticsearch's official documentation: https://www.elastic.co/guide/en/elasticsearch/reference/7.8/monitor-elasticsearch-cluster.html
+
+## Installation
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Online Licence Business & IT Editions-->
+
+1. Install the Plugin on all Centreon pollers supervising Elasticsearch ressources:
+
+```bash
+yum install centreon-plugin-Applications-Databases-Elasticsearch
+```
+
+2. Instal the Plugin-Pack 'Elasticsearch' in the " Configuration  >  Plugin Packs"  page of the Web Centreon interface
+
+<!--Offline Licenses-->
+
+1. Install the Plugin on all Centreon pollers supervising Elasticsearch ressources:
+
+```bash
 yum install centreon-plugin-Applications-Databases-Elasticsearch
 ```
 
-## Centreon Configuration
+2. Install the Plugin-Pack RPM on the Centreon server **Central**:
+
+```bash
+yum install centreon-pack-applications-databases-elasticsearch
+```
+
+3. In the *Configuration  \>  Plugin Packs*  page of the Web Centreon interface, instal the Plugin-Pack 'Elasticsearch'
 
-### Create a host using the appropriate template
+<!--END_DOCUSAURUS_CODE_TABS-->
 
-Go to *Configuration \> Hosts* and click *Add*. Then, fill the form as shown by
-the following table:
+## Configuration
+
+Still in the Web Centreon interface, go to *Configuration \> Hosts* and click *Add*. Then fill the form as shown by the following table:
 
 | Field                   | Value                        |
 | :---------------------- | :--------------------------- |
-| Host name               | *Name of the host*           |
+| Name                    | *Name of the host*           |
 | Alias                   | *Host description*           |
 | IP                      | *Host IP Address*            |
 | Monitored from          | *Monitoring Poller to use*   |
-| Host Multiple Templates | App-DB-Elascticsearch-custom |
+In the field *Templates* click on *+ Add a new entry* and then select *App-DB-Elascticsearch-custom*
```

Comment:
Prefer Bullet List, and clear instructions. 

Also specify which macros are mandatory like in https://docs.centreon.com/current/en/integrations/plugin-packs/procedures/cloud-aws-sns.html#configuration